widget/button.go

```diff
@@ -21,7 +21,8 @@ type ButtonIconPlacement int
 // ButtonImportance represents how prominent the button should appear
 //
 // Since: 1.4
-type ButtonImportance int
+// Deprecated: Use widget.Importance instead
```

Comment:
I believe that you need to have the deprecation as a new paragraph.
```suggestion
//
// Deprecated: Use widget.Importance instead
```